### ISSUE TITLE
Default filtering "Search Area" as "Current view"

### ISSUE THREAD

**Issue by MikiDi** [ENT8R/NotesReview#121]
When opening the app for the first time, a user is presented with the map at zoom level 0. An amount of notes are shown on the map. The most natural first-use pattern I imagine is zooming to an area of interest and clicking "refresh". Without having read through the entire filter modal, the results one now gets are unexpected and there's nothing to hint the user why this is happening. If you're lucky, one note may show in your current view, maybe none. At the point where I was about to give up, the "Search Area" filter option caught my view. 
To me, this experience gave a buggy impression, while the app actually works great! 
I'd advocate for setting the "Search Area" filter to "Current view" as a default. This wouldn't change anything to the behavior of the first screen load, since at zoom level 0 "Current view" and "Global" result in the same thing.

**Comment by ENT8R** [comment]
I agree that clicking on refresh and seeing no results is quite counterintuitive, but there should be a small hint at the bottom of the screen which (when clicked) zooms the map to contain all results:

<div align="center"><img src="https://github.com/ENT8R/NotesReview/assets/25306497/40954dbd-34bb-4886-a13b-a9868e875475"></div>

I would not really like to change the default search area behavior since the search within a specific area or bounding box is a little bit more expensive in terms of computation and searching in the current view by default for all users might lead to longer waiting times and an increased strain on the backend.

Maybe it would be already helpful to increase the size of the button or move it from the bottom of the screen to somewhere else, possibly in the vicinity of the other buttons at the top of the screen?

**Comment by opk12** [comment]
I would assume that most (if not all) users have a country / area of interest. What about
- auto-showing the filter dialog at launch,
- moving the area field to the top, to make it quick to reach and prominent,
- not having a default for the area field?

Reason is that the area is a special parameter: keywords and date may be unimportant, but it does not make sense to query without an area, and the default should reflect the expectation of most users.

> Maybe it would be already helpful to increase the size of the button or move it from the bottom of the screen to somewhere else, possibly in the vicinity of the other buttons at the top of the screen?

For me, the implication that `Center to the map results` means "you are not centered" therefore "the search is not restricted to this view" is too much thinking as an occasional user. Also the white background color does not stand out with respect to the land color.
